kafka-iceberg/apache-flink/README.md

```diff
@@ -0,0 +1,327 @@
+# Streaming data from Kafka to Iceberg with Apache Flink
+
+`@rmoff 2024-07-02`
+
+_👉 See the supporting blog post at https://www.decodable.co/blog/_
```

Comment:
Can we have a sentence here which describes that example on a high level. Referring to the blog post is good, but it would be nice to make the example also self-contained to some degree.

Reply:
the title of the readme is `Streaming data from Kafka to Iceberg with Apache Flink` so I think is complete as it stands.